gspread/client.py

```diff
@@ -214,6 +214,75 @@ def create(self, title):
         spreadsheet_id = r.json()['id']
         return self.open_by_key(spreadsheet_id)
 
+    def copy(self, key, title=None, copy_permissions=False):
+        """Copies a spreadsheet.
+
+        :param key: A key of a spreadsheet to copy.
+        :type title: str
+
+        :param title: (optional) A title for the new spreadsheet.
+        :type title: str
+
+        :param copy_permissions: (optional) If True, copy permissions from
+               original spreadsheet to new spreadsheet.
+        :type copy_permissions: bool
+
+
+        :returns: a :class:`~gspread.models.Spreadsheet` instance.
+
+        .. note::
+
+           In order to use this method, you need to add
+           ``https://www.googleapis.com/auth/drive`` to your oAuth scope.
+
+           Example::
+
+              scope = [
+                  'https://spreadsheets.google.com/feeds',
+                  'https://www.googleapis.com/auth/drive'
+              ]
+
+           Otherwise you will get an ``Insufficient Permission`` error
+           when you try to copy a spreadsheet.
+
+        """
+        if title is None:
```

Comment:
I believe this entire `if` block is not needed. I just tested it locally and if you omit title from the API request or pass None, Sheets API will name the copy of the spreadsheet "Copy of _ORIGINAL_NAME_".